main.go

```diff
@@ -37,6 +38,16 @@ var (
 	flagStdErr   = flag.Bool("stderr", false, "output failures to STDERR instead of to a temp file")
 )
 
+// Format a duration without the sub-second precision.
+func fmtDuration(d time.Duration) string {
+	s := d.String()
```

Comment:
This should properly handle the case of `d` being less than one second.

> // String returns a string representing the duration in the form "72h3m0.5s".
// Leading zero units are omitted.  As a special case, durations less than one
// second format use a smaller unit (milli-, micro-, or nanoseconds) to ensure
// that the leading digit is non-zero.  The zero duration formats as 0,
// with no unit.
func (d Duration) String() string {